src/infer_request.cc

```diff
@@ -151,40 +153,46 @@ InferenceRequest::SetState(InferenceRequest::State new_state)
       if (new_state != InferenceRequest::State::STARTED) {
```

Comment:
would QUEUED or PENDING be clearer - STARTED seems to imply executing ...

Reply:
I'm good with either, if I had to choose I'd pick PENDING